nep-oracles.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  NEP: <to be assigned>
+  Title: Neo oracles
+  Author: Shargon and Belane
+  Type: Standard
+  Status: Draft
+  Created: 2018-11-22
+</pre>
+
+==Abstract==
+
+The objective of this proposal is to be the first chain capable of not needing oracles to interact with non-deterministic content. Our mission is to make NEO capable of performing REST API calls and other types of non-deterministic functionalities that can be extended in the future.

```

Comment:
Hey, SharLane, why you mention REST API calls? Would it be the same as an RPC call?